packages/logo/src/ProductLogos.tsx

```diff
@@ -0,0 +1,463 @@
+import React from 'react';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { HTMLElementProps } from '@leafygreen-ui/lib';
+
+type ProductLogoProps = HTMLElementProps<'svg'> & {
+  knockout?: boolean;
+  size?: number;
+};
+
+export function CloudManagerLogo({
+  size = 18,
+  knockout = false,
+  className,
+  ...rest
+}: ProductLogoProps) {
+  return (
+    <svg
+      {...rest}
+      width={`${size}px`}
+      height={`${size}px`}
+      viewBox={`0 0 18 18`}
+      className={cx(
+        css`
+          flex-shrink: 0;
+        `,
+        className,
+      )}
+    >
+      <title>Cloud Manager Logo</title>
+      <defs>
+        <linearGradient
+          id="cloud-manager-linear-gradient"
+          x1="0.96"
+          y1="15.06"
+          x2="15.02"
+          y2="14.8"
+          gradientUnits="userSpaceOnUse"
+        >
+          <stop offset="0" stopColor="#0d6149" />
+          <stop offset="0.37" stopColor="#04a74f" />
+          <stop offset="0.65" stopColor="#00cd57" />
+          <stop offset="0.92" stopColor="#5fd590" />
+          <stop offset="1" stopColor="#7ed8a2" />
+        </linearGradient>
+        <linearGradient
+          id="cloud-manager-linear-gradient-2"
+          x1="-2.54"
+          y1="9"
+          x2="13.27"
+          y2="9"
+          xlinkHref="#cloud-manager-linear-gradient"
+        />
+        <linearGradient
+          id="cloud-manager-linear-gradient-3"
+          x1="21.41"
+          y1="15.32"
+          x2="1.11"
+          y2="15.32"
+          xlinkHref="#cloud-manager-linear-gradient"
+        />
+        <linearGradient
+          id="cloud-manager-linear-gradient-4"
+          x1="19.24"
+          y1="9"
+          x2="-3.78"
+          y2="9"
+          xlinkHref="#cloud-manager-linear-gradient"
+        />
+      </defs>
+      <path
+        className={css`
+          fill: ${!knockout
```

Comment:
Can we make a function to reap these ternary expressions, so that we don't need to nest ternaries?

Reply:
👍